### ISSUE TITLE
Offline Support

### ISSUE THREAD

**Issue by ProfessorManhattan** [sgarciac/fuego#57]
Hey, is there anyway to enable offline support for this? Would love to see this feature if it is not already available.

**Comment by sgarciac** [comment]
I don't think so. Offline support is a feature of the android/iOS and web clients.
